### ISSUE TITLE
Drop Down Bars

### ISSUE THREAD

**Issue by laurenmcguigan** [jlm323/DecameronProject#45]
@nlottig94 Matt and I are having a little trouble with the drop down bar. Matt has the basic html for it, but we need to add the css to make it work and that is where we are having trouble. If you go to decamDrop.html on the main page under web_files, that is the html if you want to take a look. But we do need help with the css. I am going to be at the study abroad event tomorrow, would you be able to help us after. I have work at 4 but I can stay till about 3. Matt is available anytime before 4. When are you free?

**Comment by jlm323** [comment]
I made a simple logo for the site but I am not going to put it anywhere until the drop down bar is figured out since I believe it will go in the SSI with the navigation/drop down bar. If you guys figure it out and want to incorporate the logo yourself, you can just put this in the css and adjust it if you need to. 

```
img.logo {
    width: 250px;
    float: left;
}
```

And then put something like this in the SSI:

```
<img class="logo" src="decamlogo.png" alt="Logo for the Decameron Website"/>
```

**Comment by laurenmcguigan** [comment]
@jlm323 The logo looks great! we should definitely add that to our Navigation bar! I will be working on the drop down bars as much as possible today, but if I get stuck I am meeting with Becca tomorrow to get it finished. Also, @mjb232 you need to make changes to the members page with your information as soon as possible so we can get that uploaded.

**Comment by laurenmcguigan** [comment]
@mjb232 I need you to give me permissions on decamDrop

**Comment by mmm202** [comment]
@laurenmcguigan we need a link on the menu bar for the members page also, I'm not sure if your already working with that and drop down menus

**Comment by laurenmcguigan** [comment]
@mmm202 I've got the drop down bars figured out! just working on some styling!

**Comment by mmm202** [comment]
@laurenmcguigan did you add the link to get to the members page?

**Comment by laurenmcguigan** [comment]
@mmm202 yes, but it needs to be uploaded to the server, and so does my ssi, but I don't have the permissions to do that yet.

**Comment by laurenmcguigan** [comment]
@jlm323 @mjb232 I need permissions on the css and decamnav!

**Comment by jlm323** [comment]
I changed the permissions for the decamnav. I can't for the css.

**Comment by jlm323** [comment]
Nevermind, I deleted the css and uploaded the new one and changed the permissions so you should be able to access it now!

**Comment by laurenmcguigan** [comment]
Thank you! @jlm323

**Comment by laurenmcguigan** [comment]
Thank you! @jlm323

**Comment by laurenmcguigan** [comment]
Thank you! @jlm323

**Comment by laurenmcguigan** [comment]
@jlm323 @mmm202 @mjb232 Hey everyone! The nav bar is up online if you want to check it out!

**Comment by jlm323** [comment]
It looks good! All we need now is a bio/write-up from @mjb232 and a short write-up of the SVG graph from @mmm202 !

I just made some minor changes on some pages like adding the footer on pages that didn't have it. Everyone make sure you pull the changes!

**Comment by mjb232** [comment]
@laurenmcguigan that looks awesome!!! Also, @jlm323 I just edited our members with my summary and picture. Ill just push the new file onto github.

**Comment by mjb232** [comment]
Soooooooooooo I have another merge conflict......fml. Ill try and figure it out

**Comment by RJP43** [comment]
@mjb232 if you can't figure it out we can fix it tomorrow

**Comment by laurenmcguigan** [comment]
@mjb232 just delete your version and send me your write-up and picture. I'm making some changes to that page right now

**Comment by RJP43** [comment]
@laurenmcguigan way to go getting the drop downs to work!!

**Comment by laurenmcguigan** [comment]
thank you! @RJP43

**Comment by laurenmcguigan** [comment]
@RJP43  Does it look okay?!

**Comment by RJP43** [comment]
@laurenmcguigan  Yea looks pretty good. You might mess with the colors and maybe the padding on the drop down lists (the gray seems a little misplaced with your site's colors and there seems to be a lot of space between each item in the drop down lists), but it all works and overall looks good. @jlm323 I like how @laurenmcguigan worked in your little graphic design. VERY COOL GUYS!!

**Comment by RJP43** [comment]
@jlm323 You might start a check list in another issue making it clear what needs finished and who is responsible. So that if someone needs help with something they can reach out.
